AdyenCard/Validators/CardSecurityCodeValidator.swift

```diff
@@ -42,4 +42,20 @@ public final class CardSecurityCodeValidator: NumericStringValidator, AdyenObser
         minimumLength = length
     }
     
+    public func validate(_ value: String) -> ValidationStatus {
+        if super.isValid(value) {
```

Comment:
Shouldn't this move a bit down the tree?
I would assume the `isEmpty` check should trigger if the value is empty - right now the parent could override it

Reply:
You may have a point but that would be changing existing logic. Previously the `isValid` function here was calling super first then doing its own checks.

Reply:
Yeah but if we change that order it would be changing existing logic. As in currently this class is only calling `super.isValid` for validation so this way we are keeping that logic first then adding the new part